src/pages/profile/edit/profile-settings/index.tsx

```diff
@@ -82,7 +84,7 @@ function ProfileSettings() {
           panelHeading={'Preferred language'}
           initial={
             <Typography variant={'sm'} color={'secondary2.main'}>
-              English
+              {convertCodeToLanguage(language)}
```

Comment:
you can put it in `useMemo` together with getting lang code

`const [langCode, language] = useMemo(() => [profile?.language || DEFAULT_LANGUAGE.code, convertCodeToLanguage(profile?.lang)], [profile?.language]);`